src/actions/loginModalActions.js

```diff
@@ -0,0 +1,13 @@
+import { loginModal } from '../action-types';
+
+const { TOGGLE_LOGIN_MODAL } = loginModal;
+
+/**
+ * Toggles the login modal
+ * @returns {obect} action object
+ */
+const toggleLoginModal = () => ({
```

Comment:
Same here. The toggle can be more generally named. If you think otherwise, it's fine.

Reply:
@Chiazokam This has been worked